skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerLogging.scala

```diff
@@ -13,6 +14,8 @@ object ProducerLogging {
   def apply[F[_]: MonadThrowable: MeasureDuration](producer: Producer[F], log: Log[F]): Producer[F] = {
 
     new WithLogging with Producer[F] {
+      // A number of chars from record's value to log when producing fails
+      private val charsToTrim = 256
```

Comment:
could we pass that as argument with default? 
I also fear that `256` could be not enough, maybe `1024`? :)

Reply:
other option - could be passing `Show`, but I'm not sure this is needed :)

Reply:
Done. As for `Show` I think it will change user API pretty much and there's not much profit in such a change